.pre-commit-config.yaml

```diff
@@ -98,13 +98,14 @@ repos:
   # Local checks for PEP headers and more
   - repo: local
     hooks:
-#      # Hook to run "check-peps.py"
-#      - id: "check-peps"
-#        name: "Check PEPs for metadata and content enforcement"
-#        entry: "python check-peps.py"
-#        language: "system"
-#        files: "^pep-\d{4}\.(rst|txt)$"
-#        require_serial: true
+      # Hook to run "check-peps.py"
+      - id: "check-peps"
+        name: "Check PEPs for metadata and content enforcement"
+        entry: "python3 check-peps.py"
```

Comment:
I don't use Windows anymore, but I can unfortunately confirm that most Python installations on Windows do not have a `python3` executable, only `python` -- and the situation is reversed on MacOS, where many Python installations do not have a `python` executable, only `python3`. So I think this will cause exactly the problems we had in CPython in https://github.com/python/cpython/pull/109891#issuecomment-1755709823 / https://github.com/python/cpython/issues/110652, unfortunately ☹️

Reply:
That's unfortunate, any ideas for how to fix it? Some things I can think of:

- The CPython discussion mentions using a `language: python` pre-commit hook, which wastefully creates a venv.
- We can use some if condition in the script like "python3 if it exists else python". But writing that in a portable way sounds like a pain.
- Use uv to do `uv run check-peps.py`. But then we have to download uv first, which also seems like overkill.